apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1429,16 +1429,28 @@ export class PortfolioService {
   private getStartDate(aDateRange: DateRange, portfolioStart: Date) {
     switch (aDateRange) {
       case '1d':
-        portfolioStart = max([portfolioStart, subDays(new Date(), 1)]);
+        portfolioStart = max([
+          portfolioStart,
+          subDays(new Date().setHours(0, 0, 0, 0), 1)
```

Comment:
This function is exactly what you need: https://github.com/ghostfolio/ghostfolio/blob/main/libs/common/src/lib/helper.ts#L201 😄

Reply:
Never mind, it's not the same:
`2022-12-31T23:00:00.000Z` (`setHours()`) vs. `2023-01-01T00:00:00.000Z` (`resetHours(new Date())`)